R/faq.R

```diff
@@ -0,0 +1,17 @@
+#' Error: Must be used within a *selecting* function
+#'
+#' @description
+#'
+#' @includeRmd man/faq/selection-context.Rmd
+#'
```

Comment:
I tried this but reverted because the automatic cross-linking didn't seem very useful. It seemed to imply that relevant information would be found in the links:

```
\seealso{
Other faq: 
\code{\link{faq-selection-context}}
}
\concept{faq}
```